nmsg/zbuf.c

```diff
@@ -139,6 +139,7 @@ nmsg_zbuf_inflate(nmsg_zbuf_t zb, size_t z_len, u_char *z_buf,
 
 	zret = inflate(&zb->zs, Z_NO_FLUSH);
 	if (zret != Z_STREAM_END || zb->zs.avail_out != 0) {
+		_nmsg_dprintf(1, "%s: inflate() failed\n", __func__);
```

Comment:
maybe output the error from inflate():
> inflate() returns ... Z_NEED_DICT if a preset dictionary is needed at this point _[djw: not possible]_, Z_DATA_ERROR if the input data was corrupted (input stream not conforming to the zlib format or incorrect check value, in which case strm->msg points to a string with a more specific error), Z_STREAM_ERROR if the stream structure was inconsistent (for example next_in or next_out was Z_NULL, or the state was inadvertently written over by the application), Z_MEM_ERROR if there was not enough memory, Z_BUF_ERROR if no progress was possible or if there was not enough room in the output buffer when Z_FINISH is used. 
 Z_DATA_ERROR is returned, the application may then call inflateSync() to look for a good compression block if a partial recovery of the data is to be attempted.